src/widgets/medications/medication-order.component.tsx

```diff
@@ -172,7 +173,7 @@ export function MedicationOrder(props: MedicationOrderProps) {
     if (props.orderEdit.orderEdit) {
       const order = props.orderEdit.order;
       setEncounterUuid(order.encounterUuid);
-      setStartDate(dayjs(new Date()).format("DD-MMM-YYYY"));
+      setStartDate(dayjs(order.startDate).format("DD-MMM-YYYY"));
```

Comment:
whtat if `startDate` is undefined

Reply:
This can't happen because I have checked if props.orderEdit is not null